File: Java/src/test/java/workshops/testingil/dirtytests/part1/UnitTests.java
package workshops.testingil.dirtytests.part1;

import org.junit.jupiter.api.Test;
import workshops.testingil.dirtytests.app.Calculator;

import static org.junit.jupiter.api.Assertions.assertEquals;
import static org.junit.jupiter.api.Assertions.assertTrue;

public class UnitTests {

    @Test
    public void test_nothing(){
        Calculator c = new Calculator();
        assertTrue(c.getDisplay() == "0");
    }

    @Test
    public void test_1(){
        Calculator c = new Calculator();
        c.press("1");
        assertTrue(c.getDisplay() == "1");
    }

    @Test
    public void test_1_and_C()
    {
        Calculator c = new Calculator();
        c.press("1");
        c.press("C");
        assertEquals(c.getDisplay(),"0");
    }
}

File: C#/NewDirtyTests/DirtyTestsWorkshop/DirtyTestsWorkshop/Exercise/Temp/UnitTests.cs
using DirtyTestsWorkshop.App;
using NUnit.Framework;

namespace DirtyTestsWorkshop.Exercise.Temp
{
    [TestFixture]
    public class UnitTests
    {
        [Test]
        public void test_nothing()
        {
            Calculator c = new Calculator();
            Assert.IsTrue(c.GetDisplay() == "0");
        }

        [Test]
        public void test_1() 
        { 
                Calculator c = new Calculator();
                c.Press("1");
                Assert.IsTrue(c.GetDisplay() == "1");
        }

        [Test]
        public void test_1_and_C()
        {
            Calculator c = new Calculator();
            c.Press("1");
            c.Press("C");
            Assert.AreEqual(c.GetDisplay(), "0");
        }
    }
}
